src/tools/Avalonia.Generators/Compiler/NameDirectiveTransformer.cs

```diff
@@ -0,0 +1,28 @@
+﻿using XamlX;
+using XamlX.Ast;
+using XamlX.Transform;
+
+namespace Avalonia.Generators.Compiler;
+
+internal class NameDirectiveTransformer : IXamlAstTransformer
```

Comment:
These AST transformers could probably be borrowed from the Avalonia main project sources. However, this is probably out of scope of this PR and could be done later.